src/main/java/seedu/duke/commands/ExchangeCommand.java

```diff
@@ -123,4 +124,28 @@ public BigDecimal parseAmount () throws InvalidNumberException {
             throw new InvalidNumberException();
         }
     }
+
+    /**
+    * Prints a message indicating if any required accounts are missing.
+    *
+    * @param ui the user interface to use for printing the message
+    * @param accounts the list of accounts to check for missing accounts
+    */
+    private void printMissingAccounts (Ui ui, AccountList accounts) {
+        try {
+            Forex exchangeRate = formatInput();
+            try {
+                accounts.getAccount(exchangeRate.getInitial());
+            } catch (NoAccountException f) {
+                ui.printMessage("You need a " + exchangeRate.getInitial() + " account!");
+            }
+            try {
+                accounts.getAccount(exchangeRate.getTarget());
+            } catch (NoAccountException f) {
```

Comment:
The standard of naming the exception variable e should be followed throughout the codebase.
```suggestion
            } catch (NoAccountException e) {
```